packages/cms-lib/errorHandlers.js

```diff
@@ -14,6 +14,8 @@ const isApiUploadValidationError = err =>
 const isSystemError = err =>
   err.errno != null && err.code != null && err.syscall != null;
 const isFatalError = err => err instanceof HubSpotAuthError;
+const contactSupportString =
+  'Please try again or visit https://help.hubspot.com/ to submit a ticket or contact HubSpot Support if the issue persists.';
```

Comment:
I think we should recommend contacting support only if the issue persists. @iamw11y any other suggestions when it comes to copy?

Reply:
I copied this directly from @iamw11y 's comment https://github.com/HubSpot/hubspot-cms-tools/issues/45#issuecomment-579284315

Reply:
Ah, cool. Sorry for the noise.